Tools/autotest/ArduCopter_Tests/MAV_LANDED_STATE_TAKEOFF/copter_mission.txt

```diff
@@ -0,0 +1,3 @@
+QGC WPL 110
+0	0	0	16	0.000000	0.000000	0.000000	0.000000	-35.363262	149.165237	584.089966	1
+1	0	3	22	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	50.000000	1
```

Comment:
could probably reduce to 30m for faster test

Reply:
Done